### PR TITLE
Made the new JavaScript SDK more prominent

### PR DESCRIPTION
**Pull request by jongio**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: README.md [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
-- The packages in [azure-sdk-for-js](https://github.com/azure/azure-sdk-for-js/tree/master/sdk) are isomorphic (can be run in the browser and on the server (node.js)) with accurate TypeScript type definitions.
-- They will be actively supported moving forward.
-- You will find any equivalent JS package in `@azure` org on npm for the existing node sdk package. For example: [azure-arm-compute](https://www.npmjs.com/package/azure-arm-compute) node sdk's equivalent JS sdk is [@azure/arm-compute](https://www.npmjs.com/package/@azure/arm-compute). 
-- _All_ the management plane sdks for the V1 Azure API (Azure Service Management) `azure-asm-*` will still be maintained from this repository. We would expect customers using these packages to move to Azure Resource Manager API packages at the earliest.
+We are excited to announce that we are building a new [Azure SDK for JavaScript](https://github.com/azure/azure-sdk-for-js), which includes both Node.js and Browser support!
```
Comment (ramya-rao-a) [inline README.md]: This here doesn't tell me that the new sdk is being built elsewhere until I click on the link or read the next couple of lines
Text like this is also used to announce a new project in the repo where the readme lives.

Consider including a line on how the new project replaces the one in this repo?

File: README.md [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
+We are excited to announce that we are building a new [Azure SDK for JavaScript](https://github.com/azure/azure-sdk-for-js), which includes both Node.js and Browser support! 
 
-**NOTE:** Apart from the noticable changes in **require/import** statements, we have tried our level best to make _lesser_ breaking changes in the API calls.
+The following will help you update your application to the new SDK and related packages.
 
-### Migration
-You can find more information on migrating your scripts from **node** sdk to **JS** sdk over [here](https://github.com/Azure/azure-sdk-for-js/blob/master/documentation/Migration.md).
+- **Packages**: 
+    - All of the management plane SDKs (`azure-arm-*`) and most of the data plane SDKs (`azure-*`), along with the rollup package [azure](https://www.npmjs.com/package/azure) in this repository, are deprecated as of **July, 2019**. They have been replaced by the [Azure SDK for JavaScript](https://github.com/azure/azure-sdk-for-js) packages.
+    - The new actively managed packages in [Azure SDK for JavaScript](https://github.com/azure/azure-sdk-for-js) support both Node.js and the Browser and ship with TypeScript type definitions.
```
Comment (ramya-rao-a) [inline README.md]:
```suggestion
    - The new actively managed packages in [Azure SDK for JavaScript](https://github.com/azure/azure-sdk-for-js) support both Node.js and Browser. They also ship with TypeScript type definitions.
```

File: README.md [hunk 3]
```diff
@@ -1,17 +1,21 @@
...
+    - You will find all equivalent JavaScript packages in `@azure` org on npm for the existing Node.js SDK package. For example, [azure-arm-compute](https://www.npmjs.com/package/azure-arm-compute) Node.js SDK's equivalent JavaScript SDK is [@azure/arm-compute](https://www.npmjs.com/package/@azure/arm-compute). 
+    - All of the management plane SDKs for the V1 Azure API (Azure Service Management) `azure-asm-*` will continue to be maintained from the Azure SDK for Node.js repository. We expect customers using these packages to move to Azure Resource Manager API packages at their earliest convenience.
+- **Breaking Changes**: Apart from the noticable changes in **require/import** statements, we have tried our best to make minimal breaking changes in the API calls.
+- **Migration**: You can find more information on migrating your code from **Azure SDK for Node.js** to the new **Azure SDK for JavaScript** [here](https://github.com/Azure/azure-sdk-for-js/blob/master/documentation/Migration.md).
```
Comment (ramya-rao-a) [inline README.md]: Link text like `here` doesn't help screenreader users much.

```suggestion
- **Migration**: See the [migration guide](https://github.com/Azure/azure-sdk-for-js/blob/master/documentation/Migration.md) to find pointers on how to migrate your code from using **Azure SDK for Node.js** to the new **Azure SDK for JavaScript** .
```

File: README.md [hunk 4]
```diff
@@ -1,17 +1,21 @@
...
+The following will help you update your application to the new SDK and related packages.
 
-### Migration
-You can find more information on migrating your scripts from **node** sdk to **JS** sdk over [here](https://github.com/Azure/azure-sdk-for-js/blob/master/documentation/Migration.md).
+- **Packages**: 
+    - All of the management plane SDKs (`azure-arm-*`) and most of the data plane SDKs (`azure-*`), along with the rollup package [azure](https://www.npmjs.com/package/azure) in this repository, are deprecated as of **July, 2019**. They have been replaced by the [Azure SDK for JavaScript](https://github.com/azure/azure-sdk-for-js) packages.
+    - The new actively managed packages in [Azure SDK for JavaScript](https://github.com/azure/azure-sdk-for-js) support both Node.js and the Browser and ship with TypeScript type definitions.
+    - You will find all equivalent JavaScript packages in `@azure` org on npm for the existing Node.js SDK package. For example, [azure-arm-compute](https://www.npmjs.com/package/azure-arm-compute) Node.js SDK's equivalent JavaScript SDK is [@azure/arm-compute](https://www.npmjs.com/package/@azure/arm-compute).
```
Comment (ramya-rao-a) [inline README.md]: While it is straight forward to find equivalent packages for the management libraries, it is not that simple for data plane libraries.

Should we consider updating the table at https://github.com/Azure/azure-sdk-for-node/blob/af611c0d8a14882fe941022194ae2786410a8d6e/README.md#azure-service-modules to include a column which points to the equivalent library/libraries?

For example, `azure-keyvault` and `azure-storage` both are split into 3 different libraries.
`azure-sb` is replaced with `@azure/service-bus` with the a whole new implementation and different api

File: README.md [hunk 5]
```diff
@@ -1,17 +1,21 @@
...
+    - The new actively managed packages in [Azure SDK for JavaScript](https://github.com/azure/azure-sdk-for-js) support both Node.js and the Browser and ship with TypeScript type definitions.
+    - You will find all equivalent JavaScript packages in `@azure` org on npm for the existing Node.js SDK package. For example, [azure-arm-compute](https://www.npmjs.com/package/azure-arm-compute) Node.js SDK's equivalent JavaScript SDK is [@azure/arm-compute](https://www.npmjs.com/package/@azure/arm-compute). 
+    - All of the management plane SDKs for the V1 Azure API (Azure Service Management) `azure-asm-*` will continue to be maintained from the Azure SDK for Node.js repository. We expect customers using these packages to move to Azure Resource Manager API packages at their earliest convenience.
+- **Breaking Changes**: Apart from the noticable changes in **require/import** statements, we have tried our best to make minimal breaking changes in the API calls.
```
Comment (ramya-rao-a) [inline README.md]: While this statement is true for management libraries, it is not true for the data plane libraries. Examples are key vault, storage and service bus.

I would suggest to remove this line altogether

File: README.md [hunk 6]
```diff
@@ -1,17 +1,21 @@
...
+- **Migration**: You can find more information on migrating your code from **Azure SDK for Node.js** to the new **Azure SDK for JavaScript** [here](https://github.com/Azure/azure-sdk-for-js/blob/master/documentation/Migration.md).
 
 ## Azure SDK for Node.js
+
+> NOTE: As of July, 2019, this repository applies to the V1 Azure API (Azure Service Management) only. All other packages will be replaced by the [Azure SDK for JavaScript](https://github.com/azure/azure-sdk-for-js). Please see the note above for more information.
```
Comment (ramya-rao-a) [inline README.md]: > All other packages will be replaced by the [Azure SDK for JavaScript]

This is not entirely accurate...

From the table at https://github.com/Azure/azure-sdk-for-node/blob/af611c0d8a14882fe941022194ae2786410a8d6e/README.md#azure-service-modules, the below libraries don't seem to have a counterpart
- azure-gallery
- azure-monitoring
- azure-scheduler

@amarzavery Any ideas on why the above don't have a counterpart in the new repo?

File: README.md (hunk 4 above)
Comment (jongio) [reply README.md]: I removed this line as it should be covered in the migration guide that is already linked.

File: README.md (hunk 6 above)
Comment (jongio) [reply README.md]: I reworded to be less specific.

**Pull request merged**